src/components/MarkerPopup/DesktopPopup.jsx

```
@@ -0,0 +1,25 @@
+const StyledPopup = styled(Popup)`
+    min-width: 300px;
+`;
+
+// DesktopPopup is a workaround for not existing lazy loading Popup as
+// react-leaflet Popup doesn't support .open() function
+export const DesktopPopup = ({ children }) => {
+    const popupRef = useRef(null);
+
+    useEffect(() => {
+        if (popupRef.current) {
+            const marker = popupRef.current._source;
```

ESLint

[error] Unexpected dangling '_' in '_source'.

(no-underscore-dangle)